test_sitemap_utils.py

```
@@ -0,0 +1,237 @@
+import unittest
+from unittest import mock
+import xml.etree.ElementTree as ET
```

Ruff (0.8.2)

`xml.etree.ElementTree` imported but unused

Remove unused import: `xml.etree.ElementTree`

(F401)